spec/controllers/leaderboard_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::LeaderboardController, type: :request do
+  subject(:api_request){ get(api_v1_leaderboard_index_path, params: params, headers: headers, as: :json)}
```

Comment:
Acordarse que los `{}` siempre llevan espacios al principio y al final (excepto cuando se usan para interpolar strings, pero eso es otro caso).
```suggestion
  subject(:api_request) { get(api_v1_leaderboard_index_path, params: params, headers: headers, as: :json) }
```

Reply:
Creo que son más de 100 caracteres, así que habría que cambiar la definición de este bloque en distintas lineas